Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -92,17 +92,72 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                     DateHelper.EndMonthUpdater(update.CallbackQuery, ref user);
                     await new SendDayCommand(mediator, telegraBotClient).SendAsync(
                         update.CallbackQuery
-                        ,"Please Select Workplace Booking Type "
+                        , "Please select end date day"
```

Comment:
it's better to leave only "select end date" (not critical)

Reply:
done